### ISSUE TITLE
TestCafe + GoJs double click does not work

### ISSUE THREAD

**Issue by KingEdwardI** [DevExpress/testcafe#6289]
### What is your Test Scenario?

Double clicking on a GoJs Canvas element.
 
### What is the Current behavior?

Double click does not call handler.
 
### What is the Expected behavior?

Double click should call the handler.
 
### What is your web application and your TestCafe test code?

https://github.com/KingEdwardI/gojs-react-testcafe-example

We are using GoJs to manage our canvas interactions and testcafe to test. Most interactions work just fine, but double clicking does not. GoJs has some custom implementation for how they handle different types of clicks (double, single, context) and doesn't base it directly off of the dom listener for `dblclick`. The code works fine when manually interacting, but not in the testcafe environment. 

Attempting the same behavior works correctly in Cypress.

I've also asked GoJs about this, they were able to provide some details about the implementation of their click handlers, but unable to help on the testcafe side of things. [see the issue here](https://forum.nwoods.com/t/double-click-not-working-in-testcafe-tests/14654)

Adjusting the test-speed does nothing, clicks are always within the 500ms threshold for double-clicking. 

This isn't browser specific, the same behavior is observed on different browsers. 

### Steps to Reproduce:

1. Clone repo
2. Run `npm start`
3. Run `npm run testcafe -- chrome test/test.spec.js`
4. Open the browser console before tests start
5. Observe that `t.doubleClick` does activate a DOM dblclick, but no `ObjectDoubleClicked` event is fired from GoJs.
 
### Your Environment details:
 
* testcafe version:                   1.14.2
* node.js version:                    14.16.0
* browser name and version: chrome90, firefox88, safari14.1.1
* platform and version:           macOS 10.15.7

**Comment by VasilyStrelyaev** [comment]
Thank you for reporting this. I have reproduced the described behavior. We will update this thread once we have any news.

**Comment by KingEdwardI** [comment]
This is still a problem, we've run into the same issue again with new tests and still cannot figure out a workaround.

**Comment by felis2803** [comment]
Hi Edward,

I couldn't reproduce the error because there is a bug in the [gojs-react-testcafe-example](https://github.com/KingEdwardI/gojs-react-testcafe-example) project:

![изображение](https://user-images.githubusercontent.com/34184692/200500616-5c94095c-53c5-4224-afd1-dd556e235f49.png)

Please correct the project or give instructions on how to get around this obstacle.

**Comment by KingEdwardI** [comment]
Sorry about that! I've updated the project to work again as well as updated all of the relevant versions. Let me know if you need anymore details/info.

**Comment by felis2803** [comment]
I'm sorry Edward, but I still can't use your repository.
First of all, I can't build it with npm:
```
npm ERR! code ENOTFOUND
npm ERR! syscall getaddrinfo
npm ERR! errno ENOTFOUND
npm ERR! network request to https://artifactory.alteryx.com/artifactory/api/npm/npm/yauzl/-/yauzl-2.10.0.tgz failed, reason: getaddrinfo ENOTFOUND artifactory.alteryx.com
```
This is not a big problem - I can use yarn. However, perhaps that is why I am faced with the second problem, which I already wrote about:
![изображение](https://user-images.githubusercontent.com/34184692/200761627-73103a00-1b88-45d6-b6cb-db8fc24a8f15.png)

Unfortunately, your latest update did not solve this problem.

**Comment by KingEdwardI** [comment]
Sorry about that, I didn't realize that the package-lock would also store the registry. I tried with yarn and ran into that same error while running.

That being said, once I got everything running using the default npm registry it all started working in this MVP repro. We're still having trouble with it on our internal project, so it seems like the issue has been fixed in general and it's something to do with how we have things set up. I'll post back here if we come across any additional & relevant info about the cause, for posterity.

Thanks for your patience and responsiveness!
